app/contracts/work_packages/base_contract.rb

```diff
@@ -473,5 +476,9 @@ def already_in_readonly_status?
     def calculated_duration
       @calculated_duration ||= WorkPackages::Shared::Days.for(model).duration(model.start_date, model.due_date)
     end
+
+    def leaf_or_scheduled_manually?
```

Comment:
Naming after an implementation feels odd.
Maybe `schedulable?` would be a better name? so it is schedulable if leaf or scheduled manually.

Reply:
renamed to `automatically_scheduled_parent?`